ocs_ci/ocs/warp.py

```diff
@@ -37,6 +37,9 @@ def __init__(self):
         self.bucket_name = None
         self.warp_cr = templating.load_yaml(constants.WARP_OBJ_YAML)
         self.host = self.warp_cr["host"]
+        # avoid failing DNS server to resolve hostname, when running on cluster with custom storage namespace
+        if config.ENV_DATA["cluster_namespace"] not in self.host:
+            self.host = f"s3.{config.ENV_DATA['cluster_namespace']}.svc"
```

Comment:
Please remove the condition and change line 39 to  `self.host = f"s3.{config.ENV_DATA['cluster_namespace']}.svc"`